airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/format_command.py

```diff
@@ -84,8 +85,10 @@ def get_dir_to_format(self, dagger_client: dagger.Client) -> dagger.Directory:
             Directory: The directory with the files to format that are not gitignored.
         """
         # Load a directory from the host with all the files to format according to the file_filter and the .gitignore files
+        modified_files = get_modified_files_local(get_current_git_revision())
```

Comment:
I think this change is sane for local run. But this code is also runs in CI. 
Can we keep the original logic for CI?
In CI we want to format the whole repo on master, and maybe on PRs too.

Reply:
@alafanechere  think I addressed your comment. Now I'm stuck on getting the tests to pass. Not sure how to pass is_local to the formatters. Is that something you could help with?